docs/governance/security-vulnerability-disclosure-policy.md

```diff
@@ -50,7 +50,7 @@ vulnerability disclosure policy explains how this process functions from the
 perspective of the project.
 
 This vulnerability disclosure policy explains the rules and guidelines for
-{==INSERT PROJECT NAME HERE==}. It is intended to act as both a reference for
+==INSERT PROJECT NAME HERE==. It is intended to act as both a reference for
```

Comment:
Instead of repeating the `project placeholder` could we put next to its first appearance the phrase _"...hereinafter referred to as 'Project,'"_ 

If we do this anyone we is going to use this text will have to replace the `==INSERT PROJECT NAME HERE==` only once